### PR TITLE
consistency in style and code in Change Office Hours

### PR DESCRIPTION
**Pull request by antoniavonto**
before:
image

after (still multiple line to scroll, but wider):
image

now similar to other edit features:
image

### CODE REVIEW

File: src/components/Profile/components/OfficeInfoList/UpdateOfficeHoursDialog/index.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import GordonDialogBox from 'components/GordonDialogBox';
+import GordonSnackbar from 'components/Snackbar';
+import { useState } from 'react';
+import userService from 'services/user';
+
+const UpdateOfficeHours = (props) => {
+  const [open, setOpen] = useState(false);
+  const [hours, setHours] = useState(props.officeHours);
+  const [snackbar, setSnackbar] = useState({ message: '', severity: null, open: false });
+  var maxCharacter = 4000;
```
Comment (amos-cha) [inline src/components/Profile/components/OfficeInfoList/UpdateOfficeHoursDialog/index.jsx:12]:
```suggestion
  const maxCharacter = 4000;
```

File: src/components/Profile/components/OfficeInfoList/index.jsx [hunk 2]
```diff
@@ -107,18 +50,23 @@ const OfficeInfoList = ({
...
           </Grid>
-        }
-      />
-    ) : null;
+        ) : (
+          `${office_hours}`
+        )
+      }
+    />
+  ) : null;
```
Comment (amos-cha) [inline src/components/Profile/components/OfficeInfoList/index.jsx:69]:
```
 const officeHours = office_hours && (
    <ProfileInfoListItem
      title="Office Hours:"
      contentText={
        myProf ? (
          <Grid container spacing={0} alignItems="center">
            <Grid item>{office_hours}</Grid>
            <Grid item>
              <UpdateOfficeHours officeHours={office_hours} />
            </Grid>
          </Grid>
        ) : (
          `${office_hours}`
        )
      }
    />
  );
```

File: src/components/Profile/components/OfficeInfoList/UpdateOfficeHoursDialog/index.jsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        open: true,
+      });
+    }
+    setOpen(false);
+  };
+
+  return (
+    <div>
+      <IconButton style={{ marginBottom: '0.5rem' }} onClick={() => setOpen(true)} size="large">
+        <EditIcon style={{ fontSize: 20 }} />
```
Comment (amos-cha) [inline src/components/Profile/components/OfficeInfoList/UpdateOfficeHoursDialog/index.jsx:35]:
```suggestion
      <IconButton style={{ marginBottom: '0.5rem' }} onClick={() => setOpen(true)} size="large">
        <EditIcon style={{ fontSize: 20 }} />
```
personal preference would be that these are stored in .css files

**Review by amos-cha — CHANGES_REQUESTED** [review]

File: src/components/Profile/components/OfficeInfoList/UpdateOfficeHoursDialog/index.jsx (hunk 3 above)
Comment (antoniavonto) [reply src/components/Profile/components/OfficeInfoList/UpdateOfficeHoursDialog/index.jsx:35]: same as other files in the same folder
image
image
image

**Review by amos-cha — APPROVED** [review]
lgtm. This file in general needs cleanup at some point though ¯\_(ツ)_/¯

**Pull request merged**